webpack.config.js

```diff
@@ -19,16 +19,29 @@ module.exports = {
         })
       },
       {
-        test: /\.(png|eot|svg|ttf|woff|woff2)$/,
+        test: /\.svg$/,
         use: [
           {
             loader: 'file-loader',
             options: {
               name: '[path]/[name].[ext]'
             }
           }
-        ]
-      }
+        ],
+        include: [new RegExp(`images\\${path.sep}emoji\\${path.sep}`)]
```

Comment:
Use simple path here instead of regexp e.g. `include: [ path.join(__dirname, "/client/images") ]`